<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkerWidget</class>
 <widget class="QWidget" name="MarkerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="idButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>#</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="bodyComboBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Head</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neck</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Up Torso</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Collar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Up Arm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Lo Arm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Hand</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Collar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Up Arm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Lo Arm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Hand</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lo Torso</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Waist</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Up Leg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Lo Leg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Heel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R. Toe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Up Leg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Lo Leg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Heel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L. Toe</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="xSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="ySpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="zSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="grabButton">
     <property name="toolTip">
      <string>Grabs the current relative location from the simulation.</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
